pkg/cmd/attestation/auth/host.go

```diff
@@ -0,0 +1,17 @@
+package auth
+
+import (
+	"errors"
+
+	"github.com/cli/go-gh/v2/pkg/auth"
+)
+
+var ErrUnsupportedHost = errors.New("The GH_HOST environment variable is set to a custom GitHub host. gh attestation does not currently support custom GitHub Enterprise hosts")
+
+func IsHostSupported() error {
+	host, _ := auth.DefaultHost()
+	if host != "github.com" {
```

Comment:
We will also have to support `ghe.com`! So we should revisit this. tbh I'm inclined to drop this check altogether.

Reply:
@phillmv : What should happen if GHES users authenticated with their Enterprise server were to use these commands?

Reply:
Hi! I was out on Good Friday.

If you're trying to verify an artifact from a public repository, it should hit dotcom. But for a repo/org on GHES, I think the API should just error out & the cli print the error returned from the API 🤔.